Assets/LeapMotion/Core/Scripts/Utils/RuntimeGizmoManager.cs

```diff
@@ -656,38 +729,115 @@ public class RuntimeGizmoDrawer {
+        }
+        else {
+          DrawCube(box.center, box.size);
+        }
+      }
+      else if (collider is SphereCollider) {
+        SphereCollider sphere = collider as SphereCollider;
+        if (useWireframe) {
+          DrawWireSphere(sphere.center, sphere.radius);
+        }
+        else {
+          DrawSphere(sphere.center, sphere.radius);
+        }
+      }
+      else if (collider is CapsuleCollider) {
+        CapsuleCollider capsule = collider as CapsuleCollider;
+        if (useWireframe) {
+          Vector3 capsuleDir;
+          switch (capsule.direction) {
+            case 0: capsuleDir = Vector3.right; break;
+            case 1: capsuleDir = Vector3.up; break;
+            case 2: default: capsuleDir = Vector3.forward; break;
+          }
+          DrawWireCapsule(capsule.center + capsuleDir * (capsule.height / 2F - capsule.radius),
+                          capsule.center - capsuleDir * (capsule.height / 2F - capsule.radius), capsule.radius);
+        }
+        else {
+          Vector3 size = Vector3.zero;
+          size += Vector3.one * capsule.radius * 2;
+          size += new Vector3(capsule.direction == 0 ? 1 : 0,
+                              capsule.direction == 1 ? 1 : 0,
+                              capsule.direction == 2 ? 1 : 0) * (capsule.height - capsule.radius * 2);
+          DrawCube(capsule.center, size);
+        }
+      }
+      else if (collider is MeshCollider) {
+        MeshCollider mesh = collider as MeshCollider;
+        if (mesh.sharedMesh != null) {
+          if (useWireframe) {
+            DrawWireMesh(mesh.sharedMesh, Matrix4x4.identity);
+          }
+          else {
+            DrawMesh(mesh.sharedMesh, Matrix4x4.identity);
+          }
+        }
+      }
+
+      PopMatrix();
+    }
+
     /// <summary>
     /// Draws a simple XYZ-cross position gizmo at the target position, whose size is
     /// scaled relative to the main camera's distance to the target position (for reliable
     /// visibility).
     /// 
+    /// Or, if you provide an override scale, you can enforce a radius size for the gizmo.
+    /// 
     /// You can also provide a color argument and lerp coefficient towards that color from
     /// the axes' default colors (red, green, blue). Colors are lerped in HSV space.
     /// </summary>
-    public void DrawPosition(Vector3 pos, Color lerpColor, float lerpCoeff) {
-      float targetScale = 0.06f; // 6 cm at 1m away.
+    public void DrawPosition(Vector3 pos, Color lerpColor, float lerpCoeff, float? overrideScale = null) {
+      float targetScale;
+      if (overrideScale.HasValue) {
+        targetScale = overrideScale.Value;
+      }
+      else {
+        targetScale = 0.06f; // 6 cm at 1m away.
 
-      var mainCam = Camera.main;
-      var posWorldSpace = matrix * pos;
-      if (mainCam != null) {
-        float camDistance = Vector3.Distance(posWorldSpace, mainCam.transform.position);
+        var mainCam = Camera.main;
```

Comment:
Another main camera that should be validated (or maybe Camera.current)

Reply:
Will look into; I thought Camera.current only has a setting when you're in a camera callback like OnPreCull but maybe gizmos are drawn during just such a callback?